starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Character from './Character';
+
+const CharacterList = props => {
+    return (
+        <div className="character-container">
+            {props.characters.map(character => (
+                <Character 
+                character={character}
```

Comment:
Clean and readable React syntax, well done.

However, I would recommend to avoid overlapping naming conventions (ie. `character={character}`).